_source/_posts/2024-02-29-3pc.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: blog_post
+title: "The End of Third-Party Cookies"
+author: edunham
+by: advocate
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: blog/3pc/social.jpg
+type: awareness
+---
+
+## What are third-party cookies?
+
+Cookies are as old as the internet. Historically, cookies were among the only options for personalizing a user's online experience and carrying their preferences from page to page. First-party cookies are issued by the web site where they're used, and third-party cookies come from other domains. 
+
+third-party cookies allow user behavior to be tracked across different sites which they visit. These cookies are now creatively misused by advertisers and trackers to exploit personal information. For the legitimate tasks which used to require third-party cookies, like social integration and caching, there are now better and more privacy-respecting technologies available. 
```

Comment:
> exploit personal information

I feel this implies their "stealing" personal information, which is not entirely accurate. They are tracking one's browser history, more-or-less "generating" data about a given person

Reply:
> creatively misused
nit: "creatively" feels positive to me, which feels like the wrong tone for the dark patterns that are used by the advertising industry.

> like social integration and caching
Why not use more targeted examples from our customers, like "federated logins and multi-brand IDPs"

> tasks
use cases?

> there are now better and more privacy-respecting technologies available
I honestly don't think this is true. Most of what's proposed right now is very early and mostly beta/behind feature flags, with active discussions in the working groups and spotty implementation across the various browsers.

Reply:
Rewrote to better capture these nuances -- thank you!